### DISCUSSION TITLE
Is there a way to see which face (part of the photo) was used in a photo with multiple persons for classification ?

### DISCUSSION THREAD

**Discussion by dgedgedge** [nextcloud/recognize#1020] [Q&A]
In photos there may be multiple persons.
So the photo should be associated to multiple persons.
I imagine that the face recognition detect the faces of persons, and then associate a tag with each face.
I also imagine that if I reallocate the photo, this information will be used by the model to do some further classifications.
As a consequence when reallocating a photo to a person it seems important to know which face was used...

Is there a way to know which part of the photo is used for the allocation ?

**Comment by dgedgedge** [comment]
May be for each person be able to show best matches ?
May be show the figure match for each photo ?
May be able to change the match rating ?

**Comment by marcelklehr** [comment]
> Is there a way to see which face (part of the photo) was used in a photo with multiple persons for classification ?

Yes. Recognize makes this information available to downstream user interfaces. The photos app simply does not show this information in its user interface. The memories app for example does have an option to show which face in the photo is tagged.
